packages/ethereum/sdk/package.json

```diff
@@ -41,6 +41,7 @@
     "@rarible/ethers-ethereum": "^0.13.69-beta.1",
     "@rarible/test-provider": "~0.5.0",
     "@rarible/web3-ethereum": "^0.13.69-beta.1",
```

Comment:
@rarible/web3-ethereum or @rarible/web3-v4-ethereum can't be in dependencies list, it's always peerDependencies (optional). And we make sure we reflect that on readme.md file

Reply:
these dependencies in devDependencies, it's not needed to include them into peerDependencies